src/interpolation/interpolate.ts

```diff
@@ -66,8 +67,40 @@ const buildUpdates = (placeholders: Placeholder[]): Request[] => {
     // ContentPlaceholder
     if (placeholder.type == 'content') {
       const contentPlaceholder = placeholder as ContentPlaceholder
+      // Image
+      if (contentPlaceholder.output.url) {
+        const output = contentPlaceholder.output
+        // Insert image
+        updates.push({
+          insertInlineImage: {
+            uri: output.url,
+            objectSize: {
+              width: output.width ? { magnitude: output.width, unit: Unit.PT } : undefined,
+              height: output.height ? { magnitude: output.height, unit: Unit.PT } : undefined
+            },
+            location: {
+              segmentId: '', // Empty means body
+              index: placeholder.position.start
+            }
+          }
+        })
+        // Remove mustaches
+        updates.push({
+          replaceAllText: {
+            replaceText: '',
+            containsText: {
+              text: placeholder.raw,
+              matchCase: true
+            }
+          }
+        })
+      }
+
       // Text
-      if (contentPlaceholder.output === `${contentPlaceholder.output}`) {
+      if (
+        contentPlaceholder.output === `${contentPlaceholder.output}` ||
+        !isNaN(contentPlaceholder.output)
```

Comment:
Why `!isNaN(contentPlaceholder.output)`?

Reply:
Here, we want to check if the result is a simple string `x === `x`` or a number `!isNaN(x)`